internal/am/job_tracker.go

```diff
@@ -115,9 +117,36 @@ func filterSavedJobs(jobs []amclient.Job, saved map[string]struct{}) []amclient.
 // ConvertJobToPreservationTask converts an amclient.Job to a
 // package_.PreservationTask.
 func ConvertJobToPreservationTask(job amclient.Job) package_.PreservationTask {
+	st, co := getJobTimestamp(job)
 	return package_.PreservationTask{
-		TaskID: job.ID,
-		Name:   job.Name,
-		Status: jobStatusToPreservationTaskStatus[job.Status],
+		TaskID:      job.ID,
+		Name:        job.Name,
+		Status:      jobStatusToPreservationTaskStatus[job.Status],
+		StartedAt:   st,
+		CompletedAt: co,
 	}
 }
+
+func getJobTimestamp(job amclient.Job) (sql.NullTime, sql.NullTime) {
+	st := job.Tasks[0].StartedAt.Time
+	ct := job.Tasks[0].CompletedAt.Time
+
+	for _, t := range job.Tasks {
+		// if current st is after the task time set that as the startedat time.
```

Comment:
Go doc comments use complete sentences, also meaning that they start with capital letter.

```suggestion
		// If current st is after the task time set that as the startedat time.
```